Makefile

```diff
@@ -12,9 +12,8 @@
 
 # Makefile for building the Admission Controller webhook demo server and docker image.
 
-AWS_ACCOUNT_ID = $$(aws sts get-caller-identity --query Account --output text)
```

Comment:
The docker image needs the fully qualified name to push successfully to remote repository.

Reply:
I've created a IMAGE_NAME variable to handle providing custom image name which can have the repository provided